src/ebay-menu-button/menu-button.tsx

```diff
@@ -49,6 +48,8 @@ const EbayMenuButton: FC<Props> = ({
     checked,
     collapseOnSelect,
     a11yText,
+    prefixId,
```

Comment:
Marko [implementation](https://github.com/eBay/ebayui-core/blob/master/src/components/ebay-menu-button/index.marko#L96) is a little bit different, should we use that same approach? They add an id in the label and use that one

Reply:
Label added after this review